internal/pkg/cli/streams.go

```diff
@@ -30,10 +31,43 @@ func NewIOStreams() *IOStreams {
 }
 
 // NewTestingIOStreams returns a IOStream and the raw bytes buffers so we can interact with them.
+// The returned bytes buffers are goroutine safe
 // Note: mostly used for testing.
-func NewTestingIOStreams() (*IOStreams, *bytes.Buffer, *bytes.Buffer, *bytes.Buffer) {
-	in := &bytes.Buffer{}
-	out := &bytes.Buffer{}
-	err := &bytes.Buffer{}
+func NewTestingIOStreams() (*IOStreams, *SyncBuffer, *SyncBuffer, *SyncBuffer) {
+	in := &SyncBuffer{}
+	out := &SyncBuffer{}
+	err := &SyncBuffer{}
 	return &IOStreams{In: in, Out: out, Err: err}, in, out, err
 }
+
+// SyncBuffer is a goroutine safe bytes.Buffer
+type SyncBuffer struct {
```

Comment:
The test for the logs command used a `bytes.Buffer` instead of writing to stderr, because now it uses two goroutines, one for the main logs and another for the event logs, this causes a race condition in the tests.